abdm/tasks/__init__.py

```diff
@@ -0,0 +1,13 @@
+from celery import current_app
+from celery.schedules import crontab
+
+from abdm.tasks.retry_failed_care_contexts import retry_failed_care_contexts
+
+
+@current_app.on_after_finalize.connect
+def setup_periodic_tasks(sender, **kwargs):
+    sender.add_periodic_task(
+        crontab(hour="0", minute="0"),
```

Comment:
its generally not recommended to use 00:00, as this might cause a spike on the requested server, lets use some random window for this

Reply:
moved this to 2.22